reports-scheduler/src/main/kotlin/org/opensearch/reportsscheduler/settings/PluginSettings.kt

```diff
@@ -95,7 +95,7 @@ internal object PluginSettings {
         var settings: Settings? = null
         val configDirName = BootstrapInfo.getSystemProperties()?.get("es.path.conf")?.toString()
```

Comment:
not related but this should be `opensearch.path.conf`

Reply:
good point, let me fix it in this PR. Otherwise we'll forget for sure